autogalaxy/profiles/light_profiles/light_profiles_operated.py

```diff
@@ -1,31 +1,20 @@
-import numpy as np
-
-import autoarray as aa
-
 from autogalaxy.profiles.light_profiles import light_profiles as lp
-from autogalaxy.profiles.light_profiles import light_profiles_linear as lp_linear
 
-from autogalaxy import exc
+# TODO : Would rather remove `is_operated` and use `isinstance_` when checking in decorator, but currently get import error.

```

Comment:
Yeah isinstance makes a lot more sense. You could import locally (i.e. inside a function) to avoid circular imports